rules/imports.js

```diff
@@ -97,6 +97,8 @@ module.exports = {
           '**/karma.conf.js', // karma config
           '**/.eslintrc.js', // eslint config
           '**/eslint-rules/**',
+          '**/.storybook/**', // storybook config
+          '**/*.stories.{js,jsx,ts,tsx}', // storybook stories
```

Comment:
I guess it is good to append `mdx` file since sometimes storybook uses itj.

```suggestion
          '**/*.stories.{js,jsx,ts,tsx,mdx}', // storybook stories
```